app/components/candidate_interface/rejection_reasons/rejection_reasons_component.rb

```diff
@@ -19,13 +19,41 @@ def link_to_find_when_rejected_on_qualifications
         "View the course requirements on #{link}.".html_safe
       end
 
+      def render_link_for_rejection_due_to_non_uk_qualification_and_no_enic
+        link = govuk_link_to('Apply for a statement of comparability from UK ENIC', I18n.t('service_name.enic.statement_of_comparability_url'))
+
+        "If you decide to apply again, think about including a statement of comparability from UK ENIC.
+        It shows training providers how your qualifications compare to UK ones. <br><br>#{link}<br><br>
+        Applications with a statement from UK ENIC are around 28% more likely to receive an offer.".html_safe
+      end
+
       def reasons
         ::RejectionReasons.new(application_choice.structured_rejection_reasons)
       end
 
       def render_link_to_find?(reason)
         reason.id == 'qualifications' && @render_link_to_find_when_rejected_on_qualifications
       end
+
+      def render_rejection_link_to_enic?(reason)
+        return false unless reason.id == 'qualifications' && reason.selected_reasons
+
+        reason.selected_reasons.any? do |nested_reason|
+          nested_reason.id == 'unverified_qualifications' && missing_enic_reference_for_non_uk_qualifications?
+        end
+      end
+
+    private
+
+      def missing_enic_reference_for_non_uk_qualifications?
+        application_form.application_qualifications
+                        .where.not(institution_country: 'GB')
+                        .exists?(enic_reference: nil)
+      end
+
+      def application_form
+        @application_form ||= ApplicationForm.find(application_choice.application_form_id)
+      end
```

Comment:
Have we considered loading the application form and qualifications at the controller level? This could be introducing an N+1 issue where we're checking the existence of qualifications multiple times over for the same Application

Reply:
That's fair - nesting is a pain. How do you feel about raising the `missing_enic_reference_for_non_uk_qualifications?` to the ApplicationForm? Ideally we'd only perform the query once. Maybe using memoization 🤷‍♂️